pkg/jx/cmd/upgrade_apps_test.go

```diff
@@ -92,8 +92,8 @@ func TestUpgradeAppForGitOps(t *testing.T) {
 	assert.Equal(t, newVersion.String(), found[0].Version)
 }
 
-func TestUpgradeAppWithShortNameForGitOps(t *testing.T) {
-	t.Parallel()
+func qTestUpgradeAppWithShortNameForGitOps(t *testing.T) {
```

Comment:
Pretty sure you didn't mean to put a `q` there. =)

Reply:
removed it :)